<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
<!--   <widget class="QFrame" name="frameMainGame">-->
<!--    <property name="geometry">-->
<!--     <rect>-->
<!--      <x>20</x>-->
<!--      <y>20</y>-->
<!--      <width>271</width>-->
<!--      <height>271</height>-->
<!--     </rect>-->
<!--    </property>-->
<!--    <property name="frameShape">-->
<!--     <enum>QFrame::StyledPanel</enum>-->
<!--    </property>-->
<!--    <property name="frameShadow">-->
<!--     <enum>QFrame::Sunken</enum>-->
<!--    </property>-->
<!--    <property name="lineWidth">-->
<!--     <number>5</number>-->
<!--    </property>-->
<!--    <property name="midLineWidth">-->
<!--     <number>10</number>-->
<!--    </property>-->
<!--   </widget>-->
   <widget class="QPushButton" name="pushButtonPlayOrPause">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widgetTime" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelTimeIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Ch</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberTime">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetMineCount" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelMineCountIcon">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Mn</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberMineCount">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>312</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Игра</string>
    </property>
    <addaction name="action_01_new_game"/>
    <addaction name="separator"/>
    <addaction name="action_02_statistics"/>
    <addaction name="action_03_params"/>
    <addaction name="action_04_change_design"/>
    <addaction name="separator"/>
    <addaction name="action_05_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_06_view_help"/>
    <addaction name="separator"/>
    <addaction name="action_07_about"/>
    <addaction name="separator"/>
    <addaction name="action_08_other_games"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_01_new_game">
   <property name="text">
    <string>Новая игра</string>
   </property>
  </action>
  <action name="action_02_statistics">
   <property name="text">
    <string>Статистика</string>
   </property>
  </action>
  <action name="action_03_params">
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
  <action name="action_04_change_design">
   <property name="text">
    <string>Изменить оформление</string>
   </property>
  </action>
  <action name="action_05_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_06_view_help">
   <property name="text">
    <string>Посмотреть справку</string>
   </property>
  </action>
  <action name="action_07_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_08_other_games">
   <property name="text">
    <string>Другие игры в Интернете</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
